src/Dom/Browser.Dom.fs

```diff
@@ -993,7 +993,6 @@ type [<AllowNullLiteral>] GlobalEventHandlers =
     abstract onpointerover: (PointerEvent -> 'Out) with get, set
     abstract onpointerup: (PointerEvent -> 'Out) with get, set
     abstract onwheel: (WheelEvent -> 'Out) with get, set
-    abstract addEventListener: ``type``: string * listener: (Event->unit) * ?useCapture: bool -> unit
```

Comment:
`GlobalEventHandlers` doesn't actually have this API, per the spec: https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers